doc/adr/0001-use-micro-frontends.md

```diff
@@ -0,0 +1,25 @@
+# 1. Use micro frontends
+
+Date: 2020-03-24
+
+## Status
+
+Proposed
+
+## Context
+
+Having the entire frontend in a monolith is the opposite of the backend architecture, where services are optional, and
+can be modified and replaced as needed. This makes it hard, if not impossible, to change or replace a part of the
+presentation layer without having to fork/throw away the whole thing.
+
+## Decision
+
+We will create separate services for individual pages and groups of pages, that will be exposed through a proxy to
+appear as a single service. The frontends are entirely independent, and can be not used or replaced entirely at will.
+
+## Consequences
+
+- A proxy (e.g. Nginx, Varnish) is required to route between the frontends.
+- Authentication knowledge will need to be shared between the frontends.
+- Each frontend will have its own assets.
```

Comment:
Should this section also mention about frontends possibly needing access to shared assets, e.g. fonts etc? Not sure if this belongs here or is a lower level detail.

Reply:
Think it's ok to leave it, we don't know how we'll do it yet and this doesn't prevent sharing.

Reply:
Agreeing on a separate ADR when that has been figured out.